ports/soem/vcpkg.json

```
@@ -1,7 +1,6 @@
 {
   "name": "soem",
-  "version": "1.4.0",
-  "port-version": 4,
+  "version-date": "2022-12-02",
```

```suggestion
  "version-date": "2023-06-09",
```
The data here refers to the date when vcpkg updated this port to ensure when we have modified it.